data-serving/data-service/src/controllers/preprocessor.ts

```diff
@@ -16,7 +16,7 @@ const createNewMetadata = (curatorEmail: string): any => {
         revisionNumber: 0,
         creationMetadata: {
             curator: curatorEmail,
-            date: Date.now(),
+            date: new Date().toISOString(),
```

Comment:
Assuming this is related to the OOM'ing -- was moving this around as a `Date` object using a bunch more memory than this approach?

Reply:
It was a schema validation failure. Schema validation wasn't being run with bulkWrite. It was easier to change this than to change the schema, cause then everywhere it's referenced in the FE would have to change.